### ISSUE TITLE
 client.isPhoneDisconnected() raise Cannot read properties of undefined (reading 'toLowerCase')

### ISSUE THREAD

**Issue by edgaru** [open-wa/wa-automate-nodejs#2554]
### Are you using the latest version of the library?

- [X] I have checked and am using the latest version of the library.

### What type of session are you experiencing this issue on?

Tested and experienced on both

### What type of host account are you experiencing this issue on?

Personal account (normal)

### Mode

My own code

### Current Behavior

This error originated either by throwing inside of an async function without a catch block, or by rejecting a promise which was not handled with .catch(). The promise rejected with the reason:
TypeError: Cannot read properties of undefined (reading 'toLowerCase')
    at Client.<anonymous> (C:\Users\user\Documents\node\whats\node_modules\@open-wa\wa-automate\dist\api\Client.js:1152:148)
    at Generator.next (<anonymous>)
    at fulfilled (C:\Users\user\Documents\node\whats\node_modules\@open-wa\wa-automate\dist\api\Client.js:24:58)
    at processTicksAndRejections (node:internal/process/task_queues:96:5)

### Expected Behavior

return true or false

### Steps To Reproduce

const isPhoneDisconnected = await client.isPhoneDisconnected();

### create() code

```markdown
wa.create({
  headless: true,
  multiDevice: false, //required to enable multiDevice support
  useChrome: true,
  qrTimeout: 0,
})
```

### DEBUG INFO

```markdown
Debug info: {
  "WA_VERSION": "2.2206.9",
  "PAGE_UA": "WhatsApp/2.2147.16 Mozilla/5.0 (Macintosh; Intel Mac OS X 11_2_2) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/89.0.4389.90 Safari/537.36",
  "WA_AUTOMATE_VERSION": "4.32.0",
  "BROWSER_VERSION": "HeadlessChrome/99.0.4844.51",
  "OS": "Windows 10",
  "START_TS": 1646851122567
}
```

### Environment

```markdown
- OS: Windows 10
- Node: 16.13.2
- npm: 8.1.2
```

**Comment by smashah** [comment]
@github-actions run

<details>
<summary>⚡ Release!  ⚡</summary>

```js
(async () => {
function exec(cmd) {
  console.log(execSync(cmd).toString());
}
//set the version type
process.env.VERS = "patch"

// Config
const gitUserEmail = "github-actions[bot]@users.noreply.github.com";
const gitUserName = "github-actions[bot]";

exec(`echo "//registry.npmjs.org/:_authToken=$NPM_TOKEN" > .npmrc`);
exec(`git config --global user.email "${gitUserEmail}"`);
exec(`git config --global user.name "${gitUserName}"`);
exec(`npm i -D`);
exec(`npm run release-ci $VERS`);

// types only package
exec('npm config set //registry.npmjs.org/:_authToken $NPM_TOKEN')
exec(`cd ./types-only && npm version $VERS && npm run build && npm publish && cd ..`);
exec(`git commit -a -m 'updated types-only package'`);
exec(`git push --force`);

//comment on the issue
var result = execSync(`npx auto-changelog -o ./tempchangelog.txt --commit-limit false --template ./compact-keepachangelog.hbs --stdout`).toString();

    await postComment(result);

//create changelog image
exec(`npm run release-image`);
exec(`git commit -a -m 'updated release-image'`);
exec(`git push --force`);
  })();
```
</details>

**Comment by smashah** [comment]
# Changelog

## 🚀 Release [4.32.2](https://github.com/open-wa/wa-automate-nodejs/compare/4.32.1...4.32.2) (2022-03-09)

- 🩹  fix #2554 [`#2554`](https://github.com/open-wa/wa-automate-nodejs/issues/2554)
- Release 4.32.2 [`ab02d38`](https://github.com/open-wa/wa-automate-nodejs/commit/ab02d38195e6fd7ef0209eb08b288061ec78d2fe)
- updated types-only package [`45850b8`](https://github.com/open-wa/wa-automate-nodejs/commit/45850b8e14baaec5216a5f265caa7ee905e2f0f0)

**Comment by smashah** [comment]
@edgaru should be fixed in latest version. thanks
